java/lang/Double/Double.hpp

```diff
@@ -591,77 +555,68 @@ namespace Java {
              */
             static double min(double a, double b);
 
-            // TODO FIXME
+            // TODO(thoangminh): Implement later
 //            /**
 //             * Returns a hexadecimal string representation of the
-//             * {double} argument. All characters mentioned below
+//             * double argument. All characters mentioned below
 //             * are ASCII characters.
 //             *
-//             * @param   d   the {double} to be converted.
+//             * @param   d   the double to be converted.
 //             * @return a hex string representation of the argument.
 //             */
 //            static String toHexString(double d);
-						
-						/**
-						* Convert double to binary 32 bit
-						* (Single-precision floating-point format
-						* In IEEE 754-2008)
-						*
-						* @param double
-						* @return string binary 32 bit of input
-						*/
-						static string doubleToBinary32StringType(double doubleInput);
-						
-						/**
-						* Convert double to binary 64 bit
-						* (Double-precision floating-point format
-						* In IEEE 754-2008)
-						*
-						* @param double
-						* @return string binary 64 bit of input
-						*/
-						static string doubleToBinary64StringType(double doubleInput);
-						
-						/**
-						* Convert binary64StringType To Double
-						* (Double-precision floating-point format
-						* In IEEE 754-2008)
-						* To Double
-						 *
-						* @param  string
-						* @return double
-						*/
-						static double binary64StringTypeToDouble(string Binary64StringTypeInput);
-						
-						/**
-						* Convert longBits To Binary64StringType
-						* (Double-precision floating-point format
-						* In IEEE 754-2008)
-						 *
-						* @param  long
-						* @return string
-						*/
-						static string longBitsToBinary64StringType(long longBitsInput);
-						
-						/**
-						* Convert from string to double
-						*
-						* @param      s   the string to be parsed.
-						* @return     a {Double} object holding the value
-						*             represented by the {String} argument.
-						* @throws     NumberFormatException  if the string does not contain a
-						*             parsable number.
-						*/
-						static Double valueOf(String stringInput);
-						
-						/**
-						* Assign value to Double variable
-						*
-						* @param  d a double value.
-						* @return a {Double} instance representing {d}.
-						*/
-						static Double valueOf(double doubleInput);
-				};
-		}
-}
-#endif  // JAVA_LANG_DOUBLE_HPP_
\ No newline at end of file
+
+            /**
+            * Convert from string to double
+            *
+            * @param      s   the string to be parsed.
+            * @return     a {Double} object holding the value

```

Comment:
Please fix comment

Reply:
Yep, We have decided to keep return String type.